examples/notebooks/vanilla-gan-tutorial.ipynb

```diff
@@ -0,0 +1,1000 @@
+{
```

Comment:
<p>Can you explain what's happening here - what is this wrapping and why user needs it, and for which callbacks? Maybe explain the concept of phases</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-team/catalyst/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>